modules/packages/SortedSet.chpl

```diff
@@ -18,7 +18,7 @@
  */
 
 /*
-  This module contains the implementation of the ``sortedSet`` type.
+  This module contains the implementation of the 'sortedSet' type.
```

Comment:
Not broken.

Given that this is right after `SortedMap`, should we strive for consistency between the two?

If so maybe `Provides the 'sortedSet' type for storing sorted unique elements.`?

Reply:
Thanks Engin.  I believe I've resolved this in https://github.com/chapel-lang/chapel/pull/19584/commits/e6f587064d1275a38fb0f6d32d1d704ab111c74e